rbac/management/management/commands/seeds.py

```diff
@@ -18,29 +18,36 @@
 import logging
 
 from django.core.management.base import BaseCommand
-from management.seeds import group_seeding, role_seeding
+from management.seeds import group_seeding, permission_seeding, role_seeding
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
 class Command(BaseCommand):
     """Command class for running seeds."""
 
-    help = "Runs the seeding for roles and groups"
+    help = "Runs the seeding for roles, permissions and groups"
 
     def add_arguments(self, parser):
         """Add arguments to command."""
-        parser.add_argument("--roles-only", action="store_true")
-        parser.add_argument("--groups-only", action="store_true")
+        parser.add_argument("--permissions", action="store_true")
+        parser.add_argument("--roles", action="store_true")
+        parser.add_argument("--groups", action="store_true")
+        parser.add_argument("--all", action="store_true")
 
     def handle(self, *args, **options):
         """Handle method for command."""
-        if not options["groups_only"]:
+        if options["permissions"] | options["all"]:
```

Comment:
Is there an explicit reason to use a bitwise operator here?

Reply:
Yea, to provide a convenient way "all" to support do all the seeding so we don't have to explicitly specify roles, groups, and permissions.

Reply:
Sorry, I miss understood, updated 👍